manifests/n/Netstumbler/Netstumbler040/0.4.0/Netstumbler.Netstumbler040.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Netstumbler.Netstumbler040
+PackageVersion: 0.4.0

```

Comment:
Version is missing in ARP, this will cause issues.

Reply:
Still learning the package manifest process. Not seeing advice on how to resolve with Google. From my very UN-trained eyes, I see it listed in the manifests on my machine.
Please advise on how to resolve.